res/res_pjsip_header_funcs.c

```diff
@@ -679,6 +814,111 @@ static int add_header(void *obj)
 	return 0;
 }
 
+
+/*!
+ * \internal
+ * \brief Implements PJSIP_HEADER 'pass' by inserting the specified header into the channel datastore.
+ *
+ * Retrieve the inheritable channel header_datastore from the session or create one if it doesn't exist.
+ * Create and initialize the list if needed.
+ * Create the pj_strs for name and value.
+ * Create pjsip_msg and hdr_list_entry.
+ * Add the entry to the list.
+ */
+static int pass_header(void *obj)
+{
+	struct header_data *data = obj;
+	struct ast_sip_session *session = data->channel->session;
+	struct hdr_list *chan_hdr_list;
+
+	ast_debug(3, "pass_header: using inheritable channel datastore\n");
```

Comment:
this debug message if it is useful should contain more information like the session name, the same applies for all added log messages - approach them as a user, make them useful/insightful to the user including what session/channel they refer to